CHANGELOG.md

```diff
@@ -1,3 +1,20 @@
+## v 0.5.0 / 2017-11-14
+
+NOTE: This release breaks backward compatibility. `statsd_exporter` now uses
+a YAML configuration file. You must convert your mappings configuration to
+the new format when you upgrade.
+
+* [CHANGE] Use YAML for configuration file
+* [BUGFIX] Fix matching empty statsd metric components #105
+* [CHANGE] Reduce log levels #92
+* [CHANGE] Removed `-statsd.add-suffix` option flag #99
+* [IMPROVEMENT] Allow help text to be customized #87
+* [IMPROVEMENT] Add support for regex mappers #85
+* [BUGFIX] Fixes for crashes with conflicting metric values #72
```

Comment:
the PR title isn't quite correct, IIRC this is about metrics with conflicting label sets. we should also note that it's still not possible to actually rely on these metrics – we just drop one or the other instead of crashing.

Reply:
which also has operational implications – if label sets change, people must restart the exporter, it is no longer restarting for them.

Reply:
You have a succinct comment we could put here?  I admit, I'm not familiar with that code.

Reply:
> Conflicting label sets no longer crash the exporter and will be ignored. Restart to clear the remembered label set.